Resources/vrecord_functions

```diff
@@ -85,6 +85,9 @@ _validate_form(){
     elif [ "${AUDIO_CODEC_CHOICE}" = "24-bit FLAC" ] && [ "${CONTAINER_CHOICE}" = "QuickTime" ] ; then
         echo "ERROR: Incompatible audio codecs and containers have been selected.\n"
     fi
+    if [ "${VIDEO_CODEC_CHOICE}" = "ProRes" -o "${VIDEO_CODEC_CHOICE}" = "ProRes (HQ)" ] && [ "${VIDEO_BIT_DEPTH_CHOICE}" = "8 bit" ] ; then
```

Comment:
`[[ "${VIDEO_CODEC_CHOICE}" =~ "ProRes" ]]` would trap both.

Reply:
I'm cautious to use that formatting in the shell that is supported by gtkdialog. If you've tested and it does, feel welcome to send a patch.

Reply:
I understand. I guess it’s Bash 3 compatible, therefore I would expect that it works in gtkdialog, but I haven’t tested so far.

Reply:
`[[ "${VIDEO_CODEC_CHOICE}" =~ "ProRes" ]]` works here. @privatezero, can you test in your environment, iirc we had reasons to avoid `[[`.

Reply:
I ran a test: it works here on Debian (on Qubes). I guess @privatezero is on the more orthodox Ubuntu.

Reply:
Can confirm that change is no bueno on my end - GTK Dialog drops down into sh, not bash for me

Reply:
Are you on Ubuntu? If so, which one?

Reply:
yup -  still running 18.04, but will probably upgrade to 20.04 soon

Reply:
I'm going to consider this issue not blocking, since the issue is with a theoretical proposal and not this PR. Let's move this into another pull request or issue if important enough to deserve it.

Reply:
No, it’s not blocking at all on my end. I’m just curious! I presume it’s the default configuration on Ubuntu, but I’ll check on 20.04.1 LTS, which I should also have on Qubes.

Reply:
FYI @dericed @privatezero : It works fine as well on the Ubuntu stations we have in production. It’s indeed a configuration issue, but I thinks that it would be an overkill to change this parameter either temporally at script’s start or definitively at script’s installation.